README_zh.md

```diff
@@ -138,6 +138,21 @@ python manage.py group [opt: --enforcer=<enforcer_name>] add <user> <role> [opt:
 python manage.py group [opt: --enforcer=<enforcer_name>] get <user> <role> [opt:<domain>]
 ```
 
+### 使用django backend
+
+django-authorization提供了一个django backend，可以将django-authorization与[Django权限系统](https://docs.djangoproject.com/en/4.2/topics/auth/default/#permissions-and-authorization)结合，实现django的权限管理。你可以在`tests/test_backend.py`中查看使用方法。
+如果你需要启用casbin backend，你需要在`settings.py`中添加如下配置：
+
+```python
+# settings.py
+AUTHENTICATION_BACKENDS = [
+    "dauthz.backends.CasbinBackend",
+    "django.contrib.auth.backends.ModelBackend", 
+    ]
+```
+
+注意，由于django权限系统的机制，您仍需要使用pycasbin的`add_policy()`方法来添加策略。
```

Comment:
Is it possible to add, modify, and delete permissions ?
```
myuser.groups.set([group_list])
myuser.groups.add(group, group, ...)
myuser.groups.remove(group, group, ...)
myuser.groups.clear()
myuser.user_permissions.set([permission_list])
myuser.user_permissions.add(permission, permission, ...)
myuser.user_permissions.remove(permission, permission, ...)
myuser.user_permissions.clear()
```

Reply:
The django user_permissions module uses its own model, so it can't to do these operations with pycasbin.